packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -700,18 +707,24 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         legendColor: this.state.color,
         shouldHighlight,
       });
-      const barHeight: number = Math.max(yBarScale(point.y), 0);
-      let adjustedBarHeight = 0;
-
-      if (barHeight <= 0) {
+      const barHeight: number = this.props.supportNegativeYValues
+        ? yBarScale(point.y) - yBarScale(0)
+        : yBarScale(point.y);
+      // Calculate threshold for minimum visible bar height
+      const minBarHeight = Math.ceil(yBarScale(this._yMax - this._yMin) / 100.0);
```

Comment:
>const minBarHeight = Math.ceil(yBarScale(this._yMax - this._yMin) / 100.0); [](http://example.com/codeflow?start=6&length=75)

minBarHeight will be measured from the baseline and not yMax - yMin

Reply:
Okay